extensions/gdx-setup/src/com/badlogic/gdx/setup/GdxSetupUI.java

```diff
@@ -575,20 +588,23 @@ public void itemStateChanged(ItemEvent e) {
 						depCounter++;
 					}
 				}
-				
+
 				for (int left = depCounter - 5; left > 1; left--) {
 					extensionPanel.add(Box.createHorizontalBox());
 				}
 
 				extensionsPanels.add(extensionPanel);
-			}
 
-			add(extensionsLabel, new GridBagConstraints(0, 8, 1, 1, 0, 0, WEST, WEST, new Insets(20, 0, 0, 0), 0, 0));
+                extensionPanel.add(showMoreExtensionsButton);
```

Comment:
Lets put this on its own line outside of the extension panel, as that will get in the way of the official extensions.